src/bloqade/emulate/codegen/emulator_ir.py

```diff
@@ -210,30 +210,6 @@ def visit_rabi(self, amplitude: Optional[Field], phase: Optional[Field]):
                         amplitude=self.waveform_compiler.emit(wf),
                     )
                 )
-        elif (
```

Comment:
This `elif` needs to be moved down because there is another `elif` that catches the case with `phase=Non` which errors on this clause.